2-mandatory/1-recipes.js

```diff
@@ -21,5 +21,17 @@
 
   You should write and log at least 5 recipes
 */
+let favoriteRecipe ={
+  title : 'Mole',
+  servings : 2 ,
+  ingredients :['cinnamon' , 'cumin' , 'cocoa' ]
+};
+
```

Comment:
do you use a linting service to make the formatting easier in vscode? https://eslint.org/